### DISCUSSION TITLE
A good outcome for the SF community!

### DISCUSSION THREAD

**Discussion by vondele** [official-stockfish/Stockfish#4233] [Show and tell]
Happy to announce:

https://stockfishchess.org/blog/2022/chessbase-stockfish-agreement/

**Comment by pschneider1968** [comment]
Wow, I'm positively surprised by this agreement!

Chessbase have done something wrong, and they acknowledged it, yet nobody has been burning bridges with this wise agreement, quite the reverse:

- the validity and strength of the GPL 3.0 has been officially acknowledged and underlined
- Chessbase will improve their internal processes to rise awareness of the value of Free Software
- Chessbase will communicate this to their customers, while advocating FOSS themselves now
- Chessbase acknowledges under 8.b) that in the best spirit of Free Software, they have to contribute something back, and will do that with their own neural nets

So this is a decision in the best interest of the Chess community, especially the free computer chess software community and the amazing Stockfish developers, because all can continue to cooperate with each other again, and with pride, and without anybody losing face with anybody else!

Congratulations that all people involved obviously had the greater good of the community in mind when reaching this agreement, and not the idea to seek damage or to punish!
